db.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+
+// A singleton to ensure we only start the database once
+// assign the MongoMemoryServer instance to mongoServer
+let mongoServer;
+const URL  ='mongodb+srv://bhumir:bhumi@cluster0.u0lqlrf.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
Replace the hardcoded MongoDB URL with an environment variable. This will enhance the security of your application and make it more flexible. You can use the `dotenv` package to load environment variables. [important]